### DISCUSSION TITLE
The evolution of developmental psychology frameworks and assessment tools (ego stages vs. cognitive stages vs. ...!)

### DISCUSSION THREAD

**Discussion by baouroux** [life-itself/community#1017] [Research]
This thread focuses on some of the fundamentals in the notion of "development" -- ontological or inner development that is.

## Overview

A quick outline of the post: Initially looking at the frameworks and assessment tools that have gained extensive following in the consulting and coaching worlds, comparisons of ego vs cognitive development are referred to. However, taking into consideration the controversies of hierarchical frameworks, a both-and proposal is presented based on my own journey in this field, for "development" and the field of developmental psychology to be seen in terms of a liminal dynamic across the 1st, 2nd and 3rd person perspectives; convergent dimensions of cognitive development (3rd person) can therefore be counterbalanced by aesthetic (1st person) perspectives, to create a deeper "Dialogos" (2nd person).

## Reflections

This post is a reference to my own inquiry with a very high-level view of the different schools of adult developmental psychology, and latest research on comparing models. Lectica is in the post-Piagetian school, building on more recent work in Dynamic Skill theory from Kurt Fischer, and model of hierarchical complexity (MHC) from Michael Commons (here Rufus can perhaps give the better overview!).

In parallel there have been important developments in 'Ego' stage development, from the likes of Robert Kegan, Bill Torbert, Suzanne Cook-Greuter etc., that seems to take a more holistic approach to capturing the meaning-making patterns of an individual, and equally presenting the ego's adult pathway in a hierarchical path - leading to more and more leadership coaching referring to 'Vertical Development'. It is upon more generally the ego development models that initiatives such as the Inner Developmental Goals were launched.

I also have a particular interest in Lawrence Kohlberg's stages of moral development, and James Fowler's stages of faith (not just in religious terms, but 'other-than-self').

Jonathan Reams and Aiden Thornton for me are two important voices in an independent and thorough critique of where we are at with the contrasting methods and frameworks of development, comparing the above as 'hard-stage' models and 'soft-stage' models. Aiden has been going deep with his PhD thesis in examining if these tools are measuring different things, and their reliability. Jonathan published a review of Aiden's work and then Aiden replied - both published in the final edition of the Integral Review, and attached here (INTEGRAL REVIEW September 2023 Vol. 18, No. 1).

Just one extract from Aiden's paper:
"If there is only an approximate correspondence between ego development stages and stages of hierarchical complexity, then this implies that hierarchical integrations do not fully – or perhaps adequately – account for the transformation of performance between successive stages. This may also relitigate considerations about the logic that underpins the ego development scale and the basis for
distinguishing between adjacent stages (Cook-Greuter, 1990; Habermas, 1979; Noam, 1993)."
...that gives a flavour as to how much is still up for debate and understanding!

All this to say, that my own inquiry (having piloted and promoted Lectica in the UK from 2009 and trained as a certified Lectica coach in 2015), leads me to reflect on how the developmental models are certainly pointing to structural aspects of our psyche (e.g. the parts), but that contrasts with an aesthetic lens of the "whole".

We are in the contentious debate from people such as Nora Bateson (seemingly categorically anti-stage) and Dave Snowden (willing to consider stages, but not hierarchically), and yet I don't hear so far an exploration of the movement BETWEEN the hierarchical (parts) and the aesthetic (whole). I've had a first go at developing this inquiry in this LinkedIn post and GDoc paper: https://www.linkedin.com/posts/john-oliver-bb01a_reframing-the-controversies-in-developmental-activity-6987746415879798784-RRX_?utm_source=share&utm_medium=member_desktop

My main thrust is holding developmental psychology frameworks in the context of 'cognitive anthropology' - recognising valid hierarchical patterns of cognition in the 3rd person, but that we can gain a lot by simultaneously holding the 1st and 2nd person. From that, we can concentrate on what are the movements between the 1st, 2nd and 3rd person perspectives, versus getting stuck on the 3rd person validity debates. Developmental psychology becomes then a grammar for our future Dialogos (taking at two levels there references from John Vervaeke!). My intuition is that the 1st person "aesthetic" is the least discussed in the developmental community....and hence my own explorations in the artistic and creative experience ([www.tyler.world](http://www.tyler.world/)).

How might the 'dynamics' between 1st, 2nd and 3rd person (in other words integrating the Transcendentals, or looking at the liminalities across the Integral Quadrants contribute to the developmental framework discussions and our "sensemaking" of poly crisis hyper objects?!

I'd love to hear your thoughts!

PS - I've just read a book by Webb Keane, called 'Ethical Life', and the structure of his book being consciously 1st, 2nd and 3rd person, echoed exactly the above...pointing perhaps in the same direction

[Jonathan Reams vol_18_no_1_reams_review_of_facing_the_complexity_gap.pdf](https://github.com/life-itself/community/files/13976504/Jonathan.Reams.vol_18_no_1_reams_review_of_facing_the_complexity_gap.pdf)

[Aiden Thornton vol_18_no_1_thornton_response_to_reams_review_of_facing_the_complexity_gap.pdf](https://github.com/life-itself/community/files/13976495/Aiden.Thornton.vol_18_no_1_thornton_response_to_reams_review_of_facing_the_complexity_gap.pdf)

**Comment by rufuspollock** [comment]
@baouroux fantastic - you are very knowledgeable 😉 Thank-you also for the links to the Aiden Thornton and Reams piece - I have started reading these.

More comments to come.

**Reply by baouroux** [reply]
Super Rufus, look forward to hearing your impressions.

I wanted to highlight Aiden's work, since there has been precious little research I feel on comparative analysis vs the inventory type research that was mentioned in the https://github.com/orgs/life-itself/discussions/963 thread.

I did post one other comparisons paper here on GitHub last year and [linked here from Tom Murray](https://drive.google.com/file/d/1DccWMZuTpCY03TXyJpXefJPyF2GyOxKk/view?usp=sharing).

AND for me, Aiden's work opens up some fundamental reflections on what it is we are hoping to measure, inviting an anthropological lens on the endeavor of measuring (ref. Tim Ingold who's been a big influence on me for the past few years). Over all these years being close to Lectica, I remained very concerned about losing a broader perspective whilst Lectica, being a robust metric,  was zooming into / isolating just one dimension. 

Even beyond Aiden's exploration of Ego stages and Cognitive stages / hard and soft stages, something yet to be really addressed is for example beyond cognition and ego, are there "Wisdom" stages? (the hint was in my title to this post as to what other dimensions might there be!).

- Wisdom as stage or emotion?
Just listening in for example to the voices such as Timothy Morton (of hyperobject fame), and his perspective of the aesthetic, the felt sense of wisdom (i.e. wisdom is an emotion more than anything else, ref. [his presentation here](https://www.youtube.com/watch?v=0oEnP2hqyAI)) is highly stimulating when trying to marry-up any notions of developmental frameworks.

**Reply by rufuspollock** [reply]
I definitely agree that Lectica focuses on one (kind) of dimension - that which can be measured with hierarchical complexity.

The issue, so far, is to find other metrics which are similarly robust for other dimensions.

In the terminology of [maps and rafts](https://github.com/orgs/life-itself/discussions/914) i was proposing the issue is that we don't have (as) good maps plus features to locate ourselves within that map (aka a 🧭) in other domains like ego development etc as we have in the areas covered by hierarchical complexity.

Take wisdom example you mention. Can that fit into hierarchical complexity and skills? To some extent maybe. But not entirely i i think.

Or take the ["waking up" / "fundamental wellbeing" locations of Jeffery Martin](https://lifeitself.org/learn/fundamental-wellbeing). These are definitely not measurable in Lectica i think. And ... are clearly v important and interesting.

**Comment by baouroux** [comment]
Fantastic resources thanks Rufus. I listened to your interview with
Jeffrey, and will go back to that material in more depth.

So I guess, it's not too controversial to say that we can seek tools that
perform some type of measurement role (for critical discernment*) in the
'Maps' category.

Talking of measurement, maybe the human relationship with technology is
that we are compelled to 'seek ourselves' through tech, or tech and
measurement as a mirror.

I am in close contact with the team at https://gowyrd.org/tech/ who are
taking on the relay from the Princeton Engineering Anomalies Lab (PEAR),
who deploy Random Event Generators in gatherings such as tantra sessions,
to measure the intensities of the collective consciousness. Is that again
another effort to measure the 'Rafts' (as a "wellbeing location"
measurement)?!

**Reply by rufuspollock** [reply]
> So I guess, it's not too controversial to say that we can seek tools that
perform some type of measurement role (for critical discernment*) in the
'Maps' category.

Exactly! And to locate you need a "map" and features on the map by which to orient yourself e.g. if you've crossed the river you are here ... (or some kind of GPS/Compass that helps you do that locating).

> I am in close contact with the team at https://gowyrd.org/tech/ who are
taking on the relay from the Princeton Engineering Anomalies Lab (PEAR),
who deploy Random Event Generators in gatherings such as tantra sessions,
to measure the intensities of the collective consciousness. Is that again
another effort to measure the 'Rafts' (as a "wellbeing location"
measurement)?!

Just to make sure we're clear on terminology: Rafts would be a technique that helps you move around the map. This example sounds more like a "compass" 🧭 i.e. a way to locate yourself on the map. (NB: often the map itself is sufficient for location because it has *features* like rivers or mountains etc which allow us to locate ourselves on the map. For example, with Martin's locations, there are certain phenomenological markers e.g. emotion is like this, memory is like this, sense of self is like this which help us locate which location you are in).

**Reply by baouroux** [reply]
Interesting, right makes sense, thanks for these comments Rufus.

**Reply by rufuspollock** [reply]
@baouroux i think for me, having a concrete example or case study to which we can apply these ideas would help a lot. e.g. here is this person and how they changed over time. Let's look at it from this, this and this perspective. That would help a lot in making this more "chewy".

**Reply by baouroux** [reply]
Yes, absolutely, having more data and case studies is a huge need for the developmental space.

One of my ambitions in the portrait filming at www.interiortruth.com), is to capture narratives of development, but in a very contrasted way to how Lectica does it (asking the subject to respond to dilemma questions). But at least Lectica capture open-ended responses, compared to the Sentence Completion Test approaches of the main ego development tools.

I have filmed quite a few people in the complexity facilitation space (from my executive coaching / organisational development consulting network), on this playlist here: https://youtube.com/playlist?list=PLdB4saAWSraFG3auUa2W8yS_sU-9XwK1-&si=yljASXNlwQ4rFN3T

And I've been anticipating from the beginning how it might be possible to "signify" these video narratives on developmental themes.

In terms of process, there is already very interesting scale I'm hoping to use, which is measuring the degree of reflection in any narrative (from Eugene Gendlin's Focusing approach) - image attached.

The choice of signification methods of the narratives on developmental scales, and the spectrum of self-signification vs expert-observer (even AI) signification, is a deep topic for another post!

My video narratives are wanting to access what are individuals' developmental edges and what is the emotional felt sense of that...rather than just capturing the cognitive.

I've written to Jennifer Garvey-Berger at Cultivating Leadership about her potential interest to fund work in this area (she uses narratives in her complexity books), and will keep you posted.

Any funding routes that the community might be able to suggest? 

![Klein Focusing EXP Scale](https://github.com/life-itself/community/assets/112642672/b38ad867-c714-42dd-97e8-3a107d8e20b9)

PS one example of developmental edge narratives is from this Cultivating Leadership associate, Akasha, sitting with the 'not knowing': https://youtu.be/AiwwjGnp-pw?si=QVtQck0do4qQesSh

**Reply by rufuspollock** [reply]
@baouroux thank you - more interesting info.

To clarify re the concrete example: I was seeking just one concrete example (it could even be made up) of a person and their life that we could then use when discussing the concepts.

Could you provide one (or two) concrete examples like this - even if fictional. Think of it like the case studies in business schools.

**Comment by matthewmccarthy11** [comment]
Thanks for the post @baouroux , it's nice again to revisit some of these ideas. It's the first time I've heard about bringing in 1st, 2nd, or 3rd person perspectives into measuring development, and although I don't know much about what you were referencing about Aiden, I think it's an important point to bring in to these discussions, since it is usually done in the 3rd person, and I just want to put in some comments as it relates to this. 

Mainly, that I am always a bit stumped when conversations about 'measurements' come up. I would like to emphasize a certain point, or question, as it relates to this. The question being: what is the possible measuring device? We assume, I think, automatically that it is something which is some kind of object or scale, in the way a thermometer measures temperature. What can it be for human development, if we are measuring a 'process'? 

I would suggest that one 'measuring device' for human development, though, is the kind of embodiment that the human being has (or maybe 'self-contact' in Steve Marchs terminology) or what it feels like to be that person, and the first person is the only way to 'measure' this.  Although this doesn't make much sense on the 3rd person scale, I think this brings in important conversations about what it means to 'measure' something, and to go back to the thermometer example, one can say that the human body also has a 'thermometer' like measuring system, in that it can regulate a certain temperature, and that this is a 'living or natural measurement' device, and that a thermometer can also measure temperature, but is not a living measurement device.  

So, if there are two kinds of measuring devices (just a hypothesis), one which happens by life itself (!) and one which happens through objects or scales that measure (thermometers), would this distinction make any sense when trying to consider 'human development', a lived process? Another important reason I bring this up is because from my experience, we can to some degree 'measure', after a five minute conversation with someone, 'how developed' a person is, or at least be somewhat close.

**Reply by baouroux** [reply]
Super Matthew, thanks for your reflections and working through the different perspectives. We're at the heart here I feel of a fundamental dilemma for the human condition, both as individuals and collectives - our approximate / default reactions to which lead us (sleepwalking perhaps?) into reducing our meaning-making through materialism (i.e. the "Good Life" being collapsed to scientific principles, which includes the impulse to measure everything).

I include in this the efforts of AI engineers to try to code every aspect of human artefacts, the latest ambitions of which are to code human behaviour - in other words, having AI abstract, measure and codify human culture.

From my 7 week course last year with JF Martel (of the Weird Studies Podcast) on "Art in the Age of AI", we had Jacob Foster from UCLA/Santa Fe Institute present to us on AI's ambition to codify human psychology / human culture, from which [I wrote this article on LinkedIn](https://www.linkedin.com/posts/john-oliver-bb01a_a-complexity-and-cognitive-sciences-lens-activity-7150064257181634560-Olqt?utm_source=share&utm_medium=member_desktop), and I thought it a brilliant insight from the course that the engineers and investors in SIlicon Valley compulsive pushing of the horizons of AI into codifying culture, to be akin to the "aesthetic" principles of the Dada art movement. For some reason, there is a default assumption that to measure and codify even the most complex emergent factors such as human culture (or human development) is central to a universal social good. 

AI codifying of religions and faith traditions will surely be on their list at some point!

All this to say, that as a somewhat indirect reply to your message (reaching for the big picture), is that over the years I am always brought back to the Good, the True and the Beautiful, when tackling the kind of dilemma you raise in your message. Ken Wilber's Integral movement had such an impact since he was re-energising the value of the ancient Transcendentals for our meaning-making. A whole generation of baby-boomers (and people like me from the 60s!) had an ah-ha moment; now I can look at the world through these three lenses.

I would propose a liminal approach where the thermometer (the True = convergent, 3rd person) is inseparable from the aesthetic, individual interpretation reading of the temperature IN CONTEXT (the Beautiful = divergent, 1st person), from which our collective meaning-making (the Good, 2nd person) arises.

I don't seem to hear this aspect of perspective coordination in applying the Transcendentals / the Integral, to our meaning making in the meta-modern discourse....but am trying my own experiments to bring it into a conscious practice. 

I don't either want to be anti-AI in my references above, only to evoke the ever growing importance of how we place AI into our meaning-making in a balanced, Integral, way.

**Comment by baouroux** [comment]
[2013_0710_ITC_Lectica_ClearImpact.pdf](https://github.com/life-itself/community/files/14730749/2013_0710_ITC_Lectica_ClearImpact.pdf)
Right, got it...reference material that is again quite scarce from my
experience!

Since so many of the frameworks are deployed as more momentary assessments
(for coaching) vs longitudinal research studies.

I personally only have Lectica related material, mostly private client
narratives. Although this is now 10 years old, I attach a paper that
Lectica published looking at specifically longitudinal shifts...page 15
gives the "measured shift" from coaching and facilitation. The actual
narrative content could I guess be requested from Lectica, within the
constraints of what permissions they have..

Bill Torbert's Action Inquiry book is one resource that springs to mind
with case study shifts of executive's thinking.

Otherwise, I'll continue to reflect on possible sources for case study type
material..

**Reply by rufuspollock** [reply]
@baouroux thank-you (as always) for the knowledgeable response.

I note there's no file attached here - i think you may need to post it on the web interface of the discussion thread.

**Reply by baouroux** [reply]
Thanks for the spot on that Rufus - I just edited the post with hopefully the file attached correctly!

**Comment by asimong** [comment]
Slow in catching on here, but I've just recognised the personal significance of **belonging**, and very curious now how to frame belonging in terms of different positions (or stages, if you don't mind that term) which may (or may not) exist on a developmental scale. I'm interested both in the individual subjective sense of belonging (about which there seems to be tons of literature, particularly around childhood and school etc.) and in the collective sense of how much different members "belong". Obviously, these interact. It's not a question of one or the other, but both-and systemically.

**Reply by asimong** [reply]
I've now written a pair of pieces on belonging, [a more personal one](https://wiki.simongrant.org/doku.php/d:2024-03-22) and [one more directly addressing stages](https://wiki.simongrant.org/doku.php/d:2024-03-24).

**Reply by baouroux** [reply]
Very thoughtful pieces here Simon, great to read them...belonging has to be one of the most insightful lenses on the human condition and meaning making. Nice to see your reference to Theodore Zeldin, who I was in touch with a good few years ago - his observations of happiness being fundamentally rooted in the relativity of the collective, were ah-ha moments for me.

Then of course, as a contrast, we have the oscillations between identity being individually oriented and collectively oriented from the developmental frameworks - we get to know ourselves by how we break away from any sense of belonging or any collective. How does that dynamic continue throughout our lives?

I'm also intrigued by the notion of polarities/oscillations between evolution (unfolding, development) and involution (Peter Merry from Ubiquity University who I've been working with, has written a brilliant PhD thesis on the core notion their relationship as Volution. Here the involution evokes for me the "belonging" to the transpersonal.

A salute to your efforts with ReGen and CHOICE.

**Comment by catherinet1** [comment]
An example of an inner development self-assessment framework and tool which a colleague just shared. Interesting from a quick look, although not sure how rigorous it is.

UZAZU Embodied Intelligence Self-Assessment: https://www.uzazu.org/sign-up-for-the-embodied-intelligence-free-self-assessment/ 

![Screenshot 2025-01-06 at 19 45 07](https://github.com/user-attachments/assets/e5d0e603-6d1a-4d64-b15a-205b934e1f1b)

from what i gather so far, based on a questionnaire, it plots your tendency to be in states of under-activation, balanced activation, or over-activation across 2 dimensions: "1) Who or what we put our attention on—with its two poles of Self-Focused & Other-Focused and 2) How we process and respond to that stimulus—with its two poles of Sensing & Acting"

**Reply by catherinet1** [reply]
Something i find interesting about it is how it depicts how over-activation of each of the 4 diagonal dimensions or characteristics (agency, connection, collaboration, inner experience) is just as undesirable as under-activation. i.e. balance is desirable, not linear increase in any certain skill, capacity, or quality

Their description of the different state activations:


>**BALANCED ACTIVATION: A skillful, appropriate, healthy response to situations**
Balanced Activation allows us to stay pleasantly calm, cool, collected, and connected. When we’re in a balanced-activation state, we can respond to each situation or environment in the most aligned, appropriate way, resulting in more joy, ease, and flow. As you can imagine, the HIGHER your scores in these balanced-activations of states, the happier and more satisfying your life tends to be!
>
>**OVER-ACTIVATION: Doing too much, going too far, being excessive in something**
Over-Activation causes us to do too much, or go too far into a particular state of embodiment. When we’re in an over-activation state, we might feel things too intensely, overthink things, or do things too forcefully or compulsively. Having LOWER scores in the over-activated states leads to significantly less stress & overdrive, more ease & flow, and less strain on your whole system.
>
>**UNDER-ACTIVATION: Not going far enough or doing enough, feeling insufficient, blocked, or disconnected**
Under-Activation makes us not move far enough into the appropriate or desired alignment. When we’re in an insufficient state, we might feel physically or emotionally drained or depleted, or we might feel stuck. blocked, or held back in what we’re able to do. Having LOWER scores in the under-activated states leads to significantly more energy & vibrance, less stuckness & stagnation, and more fullness, empowerment, and inspiration in your life.

**Reply by catherinet1** [reply]
Apparently research based (apparently its development has been part-funded by some government research grants). https://www.uzazu.org/the-uzazu-story/

**Reply by asimong** [reply]
I tried it out. Interesting, yes. I do wonder, though, these self-assessment questionnaires, what do they reveal? Self-image? I would find it far more interesting if people were able to get into collective assessment — I don't know of any sites that offer this to the general public, and anyway where would you get the others to fill them in? I'm reminded of Burns's poem [To a Louse](https://en.wikipedia.org/wiki/To_a_Louse)
